SW_Flow_lib.c

```diff
@@ -282,7 +282,7 @@ void infiltrate_water_high(double swc[], double drain[], double *drainout, doubl
 
 	int i;
 	int j;
-	double d[nlyrs];
+	double d[OVER_SIZE] = {0};

```

Comment:
Why not use existing constant for this specific purpose of soil layer number, i.e., `MAX_LAYERS`, if we want here a constant array size?